### ISSUE TITLE
Problem with display Excalidraw in dark mode with minimal theme in obsidian 0.16+

### ISSUE THREAD

**Issue by dariuszkowalski-com** [zsviczian/obsidian-excalidraw-plugin#817]
**Describe the bug**
Problem with obsidian dark, excalidraw light:

BAD: turn on dark mode in the obsidian, Excalidraw in light mode:
![image](https://user-images.githubusercontent.com/37656900/191424719-7833e8be-0510-4e96-abac-0ba328cf690a.png)

OK turn on dark mode in the obsidian, Excalidraw in dark mode: 
![image](https://user-images.githubusercontent.com/37656900/191425028-00b466ed-01b2-4219-9484-d6dc5dc13768.png)

OK When I change Obsidian to light mode, Excalidraw in light mode looks that:
![image](https://user-images.githubusercontent.com/37656900/191424894-8edf6a48-e0f7-4b16-bac6-494332659975.png)

OK  Obsidian to light mode, Excalidraw in dark mode looks that:
![image](https://user-images.githubusercontent.com/37656900/191425280-158fbcd5-7676-4a6a-a052-d3e5ebc38f1d.png)

**To Reproduce**
I have got the newest minimal theme installed (6.0.5)

**Expected behavior**
it should looks like in default theme
![image](https://user-images.githubusercontent.com/37656900/191425630-bb00ecba-081e-41dc-ad39-b5402c409a21.png)

**Environment (please complete the following information):**
 - OS including version: Windows 10
 - Plugin version:  1,7.20
 - Obsidian version:  0.16.0 and newer

**Comment by zsviczian** [comment]
Yes, I am aware of the issue, but haven't had time to dig deeper into it. I'll try to get to this eventually... however I recommend also raising an issue with the minimal theme.

**Comment by insecurejezza** [comment]
You can add this CSS snippet as a superficial fix while you wait for an official fix:

body .excalidraw, body .excalidraw.theme--dark {
    --island-bg-color: #F5F5F5;
}
